src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
Comment #2 

So instead of passing a prop to hide the header, it might just be simpler to use `navigation.reset` to [`reset`](https://reactnavigation.org/docs/navigators/navigation-actions#Reset) the current navigation stack instead of pushing another route on top. That way there shouldn't be a navigate back functionality at all and we wouldn't have to workaround trying to pass a manual prop to hide it.

Let me know what you think @Antoine38660 :)